app/models/assignment.rb

```diff
@@ -338,7 +337,7 @@ def update_remark_request_count
 
   def total_criteria_weight
```

Comment:
I was thinking about deleting it in my next pull request because until this point we are showing a total weight in the table that includes the list of criteria, but this will change in the next pull request when we merge the UI for rubric and flexible criteria.